.github/workflows/lint_prettier.yml

```diff
@@ -0,0 +1,17 @@
+name: Prettier and Eslint Test
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    name: Prettier ans Eslint Test
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install packages
+        run: yarn install
+      - name: Prettier
+        run: yarn canvas:prettier
+      - name: EsLint
+        run: yarn canvas:eslint
```

Comment:
eslint and prettier should not run specifically in project canvas.
They should also run for other files (project extender...)
So they should run from the root.
Can you also split these checks? (them taking too long is not a problem for now. That can be solved with caching and using a good pipeline structure)